alliance_auth/settings.py.example

```diff
@@ -440,14 +441,6 @@ LOGGING = {
             'handlers': ['log_file', 'console'],
             'level': 'DEBUG',
         },
-        'sigtracker': {
-            'handlers': ['log_file', 'console'],
-            'level': 'DEBUG',
-        },
-        'optimer': {
-            'handlers': ['log_file', 'console'],
-            'level': 'DEBUG',
-        },
```

Comment:
I have no idea. I am still too new to working in git to see where it was removed. Can I have done something wrong when trying to merge master into my fork maybe?